app/js/components/cohorts/cohortModal.js

```diff
@@ -68,13 +69,22 @@ class ActionsComponent extends Component {
                             name :'', 
                             description: ''
                     }));
+                    
                     $('#myCohort').modal('hide');          
                 });
         } else {
             this.setState({ error: "all fields are required" });
         }
     }
 
+    onSave(event) {
+        event.preventDefault();
+        const {name, description} = this.state;
+        if(name && description) {
```

Comment:
@dkayiwa With the help of @andela-jomadoye, this has been fixed. It is ready for testing  [here](https://modules-refapp.openmrs.org/openmrs/owa/cohortbuilder/index.html#/)